docs/translations/api-docs/date-pickers/time-field-pt.json

```diff
@@ -3,7 +3,7 @@
   "propDescriptions": {
     "ampm": "12h/24h view for hour selection clock.",
     "autoFocus": "If <code>true</code>, the <code>input</code> element is focused during the first mount.",
-    "color": "The color of the component. It supports both default and custom theme colors, which can be added as shown in the <a href=\"https://mui.com/material-ui/customization/palette/#adding-new-colors\">palette customization guide</a>.",
+    "color": "The color of the component. It supports both default and custom theme colors, which can be added as shown in the <a href=\"https://mui.com/material-ui/customization/palette/#custom-colors\">palette customization guide</a>.",
```

Comment:
nit: this change seems unrelated. If it was intentional 🆗

Reply:
It was intentional :+1: